ig/README.md

```diff
@@ -32,3 +33,24 @@ A Dockerfile is provided that can be used to generate the Implementation Guide.
 1. The container also pre-generates the documentation.  This process could make the image build take 10-15 minutes
 1. All of the generated documentation should be in the `/trusted-intermediary/output` directory
 1. If building locally, you may want to add a `--no-cache` to the `docker buildx build` command so that the documentation generation is not cached
+
+## Using Docker for local development
+
+Provided is a Dockerfile suitable for running the implementation guide documentation generation without the need to install all the tooling locally.
+The Docker image needs to be built and then run with an appropriate `mount` flag in order to mount your local `ig` directory into the running Docker container.
+This will allow bi-directional writes so that the Docker container can generate the documentation, and as a developer you can view the generated documentation easily.
+A developer will also be able to update any input files for the documentation and re-run the documentation generation without any intermediate steps.
+
+1. In a terminal, navigate to this `ig` directory
+2. run `docker buildx build -t ig-local-dev -f Dockerfile-local-dev .` to build the Docker container
+3. run `docker run -it --mount type=bind,source="$(pwd)",target=/trusted-intermediary ig-local-dev bash` to run the container and use an interactive `bash` shell inside the container
+4. Inside the container environment it should default to a working directory of `/trusted-intermediary`; this is bound to the `ig` directory in your local git repo
+5. Periodically update the IG Publisher by running `./_updatePublisher.sh`
```

Comment:
Periodically?  Is there a better set of rules for when and why we would need to run this?

Reply:
It seems like the publisher is updated multiple times per week.  Maybe a better solution is to include a trigger that runs it automatically after the container is started and then again once per day?

Reply:
README will be updated with more details, automating this will be deferred to a different backlog task